src/main.go

```diff
@@ -98,7 +99,19 @@ func GetGpuProductNames() {
 	}
 }
 
+func GetNodeName() {
+	nodename, err := exec.Command("uname", "-n").Output()
+	if err == nil {
+	    gNodeName = string(nodename)
+	    gPod = string(nodename)
```

Comment:
Why do we need `gPod` and `gNodeName` - what is `gPod`' supposed to be, is it always same as `gNodeName`?